src/JsonRpcEngine.ts

```diff
@@ -42,6 +44,22 @@ export interface JsonRpcMiddleware<Params, Result> {
 const DESTROYED_ERROR_MESSAGE =
   'This engine is destroyed and can no longer be used.';
 
+export type JsonRpcNotificationHandler<Params> = (
+  notification: JsonRpcNotification<Params>,
+) => void | Promise<void>;
+
+interface JsonRpcEngineArgs {
+  /**
+   * A function for handling JSON-RPC notifications. A JSON-RPC notification is
+   * defined as a JSON-RPC request without an `id` property. If this option is
+   * _not_ provided, notifications will be treated the same as requests. If this
+   * option _is_ provided, notifications will be passed to the handler
+   * function without touching the engine's middleware stack. This function
+   * should not** throw or reject.
```

Comment:
Hmm, looks like the `**` before "should" disappeared. Maybe put italics/bold around "not"?